src/routers/v1/PlacesAutocompleteRouter.js

```diff
@@ -33,12 +30,12 @@ class PlacesAutocompleteRouter extends ApplicationRouter<string> {
     // not in cache
     const options = {
       method: 'GET',
-      url: 'https://maps.googleapis.com/maps/api/place/autocomplete/json',
+      url: Constants.GOOGLE_AUTOCOMPLETE_URL,
       qs: {
         input: query,
         key: process.env.PLACES_KEY,
         location: '42.4440,-76.5019',
-        radius: 24140,
+        radius: Constants.RADIUS,
```

Comment:
Can we make this more descriptive like `Constants.AUTOCOMPLETE_RADIUS` or something